_docs/_latest/setup/install.md

```diff
@@ -24,111 +24,14 @@ organization Cloud IAM policy.
 
 ## Setting up Forseti Security
 
-The installer automatically determines setup information, generates a deployment
-template, and creates a Forseti deployment.
+Users can install Forseti using Terraform by clicking the button below.
 
-### Terraform installation
+[![install forseti using cloud shell tutorial]({{site.url}}/images/docs/setup/install_button_using_cloud_shell_tutorial.png)](http://www.google.com)
```

Comment:
Link should be: https://github.com/forseti-security/terraform-google-forseti/tree/master/examples/install_simple

Reply:
Make the changes that morgante suggested.